WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreSetupProgressView.swift

```diff
@@ -54,13 +54,18 @@ struct StoreSetupProgressView: View {
                 }
 
                 Button(Localization.hideStoreSetupListButton) {
-                    showingHideStoreSetupListAlert = true
+                    if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.dynamicDashboard) {
+                        hideTaskListAction()
+                    } else {
+                        showingHideStoreSetupListAlert = true
+                    }
                 }
                 .renderedIf(isHideStoreOnboardingTaskListFeatureEnabled)
             } label: {
-                Image(uiImage: .ellipsisImage)
-                    .flipsForRightToLeftLayoutDirection(true)
-                    .foregroundColor(Color(.textTertiary))
+                Image(systemName: "ellipsis")
```

Comment:
nit: I noticed we already have in the codebase an image from Gridicons called `ellipsis`. Is there any reason why we are using the system one? Is it part of Joe's design?

Reply:
Yes, this was stated explicitly by Joe in the design:

<img width="419" alt="image" src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/83e97dd8-70e7-4f8a-b195-95b8e7c523f9">

Also, system images can be scaled properly so when a device uses a large font, the icons can be scaled accordingly too. This is a recommended use of icons by Apple.